app/views/to_do_lists/_no_matches.html.haml

```diff
@@ -1,3 +1,4 @@
-.empty
-  = image_tag 'blank_lists.jpg', size: '200x206', alt: 'No lists icon'
-  %h2= t 'search.no_results'
\ No newline at end of file
+#lists-container
+  .empty
+    = image_tag 'blank_lists.jpg', size: '200x206', alt: 'No lists icon'
+    %h2= t 'search.no_results'
```

Comment:
@vlasiak, you can DRY duplicated code of the view into a partial and use in `_no_lists` and `_no_matches` partials, if you want. 
```haml
#lists-container
  .empty
    = image_tag 'blank_lists.jpg', size: '200x206', alt: 'No lists icon'
```